gordo_components/cli/workflow_generator.py

```diff
@@ -126,6 +127,13 @@ def workflow_generator_cli(**ctx: dict):
     context: Dict[Any, Any] = ctx.copy()
 
     yaml_content = wg.get_dict_from_yaml(context["machine_config"])
+    log_level = (
+        yaml_content.get("globals", {})
+        .get("runtime", {})
+        .get("log_level", os.getenv("GORDO_LOG_LEVEL", "INFO"))
```

Comment:
why not the one from the context? With this one can set the loglevel using the CLI  ``--log-level`` but then get, or not get, it set on the workflow depending on what the environment variable is.

Reply:
I guess it should be possible, but I am not sure how to do it correctly with click. It is not part of the ctx in the workflow, and I have not been able to pass/retrieve it from the CLI context. Perhaps @milesgranger knows something about this.

Reply:
You could possibly store a [local config file,](https://click.palletsprojects.com/en/7.x/utils/#finding-application-folders) which could then be read by the sub-CLIs.  Maybe wrap things up in a decorator and ensure is transient.

Reply:
You have tried to use `@click.pass_context`?

Reply:
@epa095 I have talked to our American friend @milesgranger about it, and he also suggested passing the context. I tried it before, but probably did not pass it far enough down the stream to be able to retrieve it in the workflow_generator. 
@ryanjdillon Good suggestion, but I think if we can use click we should, it's probably just me doing it wrong :man_shrugging: